recipe_scrapers/drinkoteket.py

```diff
@@ -0,0 +1,53 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class Drinkoteket(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "drinkoteket.se"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return 1
```

Comment:
Thank you, @Pebkac03!

The only quibble I have is this `return 1`.  I guess it makes sense that most/all of the recipes will be for a single instance of whatever cocktail is being prepared.  Could you check for counter-examples even so?

Reply:
Thanks, you're right. Found that some recipes where for two servings. Will probably work on it tomorrow.